lib/block_view.go

```diff
@@ -1773,6 +1780,54 @@ func (bav *UtxoView) _connectBasicTransfer(
 		}
 
 		utxoOpsForTxn = append(utxoOpsForTxn, newUtxoOp)
+		// Certain transaction types spend more DESO than input specified
```

Comment:
Instead of computing this here, could we just pass in an additionalSpend uint64 to connectBasicTransfer? And then compute these in the corresponding connectCreatorCoin, connectNFT, connectDAOCoinLimitOrder? Downside is would be updating the connectBasicTransfer func signature which is used in a bunch of places. Upside is it better encapsulates each txn types logic.

Reply:
Also could avoid storing the filledOrders on the utxoView in that case.